Library/Formula/watch-sim.rb

```diff
@@ -0,0 +1,18 @@
+class WatchSim < Formula
+  homepage "https://github.com/alloy/watch-sim"
+  url "https://github.com/alloy/watch-sim/archive/1.0.0.tar.gz"
+  sha256 "138616472e980276999fee47072a24501ea53ce3f7095a3de940e683341b7cba"
+  head "https://github.com/alloy/watch-sim.git"
+
+  depends_on :xcode => "6.2"
+
+  def install
+    system "make"
+    bin.install "watch-sim"
```

Comment:
A `make install` would be great if you could submit an issue/PR for a future release? Thanks!